File: src/main/java/no/nav/veilarboppfolging/controller/response/TilordneVeilederResponse.java
package no.nav.veilarboppfolging.controller.response;

import lombok.Data;
import lombok.experimental.Accessors;
import no.nav.veilarboppfolging.controller.request.VeilederTilordning;

import java.util.List;

@Data
@Accessors(chain = true)
public class TilordneVeilederResponse {
    String resultat;
    List<VeilederTilordning> feilendeTilordninger;
}

File: src/main/kotlin/no/nav/veilarboppfolging/controller/response/TilordneVeilederResponse.kt
package no.nav.veilarboppfolging.controller.response

import no.nav.veilarboppfolging.controller.request.VeilederTilordning


data class TilordneVeilederResponse(
    val resultat: String,
    val feilendeTilordninger: List<VeilederTilordning> = emptyList()
)
